src/main/java/soconnect/ui/PersonCard.java

```diff
@@ -58,73 +59,92 @@ public PersonCard(Person person, int displayedIndex, Logic logic) {
-        if (other == this) {
-            return true;
-        }
-
-        // instanceof handles nulls
-        if (!(other instanceof PersonCard)) {
-            return false;
-        }
-
-        // state check
-        PersonCard card = (PersonCard) other;
-        return id.getText().equals(card.id.getText())
-                && person.equals(card.person);
-    }
-
     /**
      * Sets the attributes in the order required.
      */
     private void setAttributes() {
         int[] order = filterAttributes(orderAttributes());
-        FlowPane[] flowpanes = new FlowPane[5];
+        FlowPane[] flowpanes = new FlowPane[NUMBER_OF_CUSTOMISABLE_ATTRIBUTES + 1];
         flowpanes[order[0]] = attributeA;
         flowpanes[order[1]] = attributeB;
         flowpanes[order[2]] = attributeC;
         flowpanes[order[3]] = attributeD;
 
         if (flowpanes[0] != null) {
             flowpanes[0].getChildren().add(new Label(person.getAddress().value));
+            flowpanes[0].getChildren().forEach(label -> label.setStyle(
+                    "-fx-font-size: 12;-fx-font-family: \"Segoe UI Semibold\";"));
         }
         if (flowpanes[1] != null) {
             flowpanes[1].getChildren().add(new Label(person.getEmail().value));
+            flowpanes[1].getChildren().forEach(label -> label.setStyle(
+                    "-fx-font-size: 12;-fx-font-family: \"Segoe UI Semibold\";"));
         }
         if (flowpanes[2] != null) {
             flowpanes[2].getChildren().add(new Label(person.getPhone().value));
+            flowpanes[2].getChildren().forEach(label -> label.setStyle(
+                    "-fx-font-size: 12;-fx-font-family: \"Segoe UI Semibold\";"));
         }
         if (flowpanes[3] != null) {
             person.getTags().stream()
                     .sorted(Comparator.comparing(tag -> tag.tagName))
-                    .forEach(tag -> flowpanes[3].getChildren().add(new Label(tag.tagName)));
+                    .forEach(tag -> flowpanes[3].getChildren().add((new Label(tag.tagName))));
+            flowpanes[3].getChildren().forEach(label -> label.setStyle("-fx-background-color: #3e7b91;"
+                    + "-fx-font-size: 11;-fx-background-radius: 2;"
+                    + "-fx-border-radius: 2;-fx-padding: 1 3 1 3;"));
         }
     }
 
     /**
-     * Filters the attributes based on what attributes were chosen to be hidden.
+     * Generates the order of the attributes based on the order set by the user.
+     *
+     * @return The required order.
+     */
+    private int[] orderAttributes() {
+        int[] order = new int[NUMBER_OF_CUSTOMISABLE_ATTRIBUTES];
+        String orderStr = logic.getAttributeOrder();
+        String[] orderArr = orderStr.trim().split(">");
+
+        if (orderArr.length != NUMBER_OF_CUSTOMISABLE_ATTRIBUTES) {
+            //Returns default order when the orderStr is not in correct format.
+            order = new int[]{3, 2, 1, 0};
+            return order;
```

Comment:
I think it might be better to use the `DEFAULT_ORDER` that you have declared in `GuiSetting.java`, and then use `convertToIndex` method to convert the attributes to the index to avoid these magic numbers.

Also this makes the code more extensible since we only have to modify the `DEFAULT_ORDER` and the `convertToIndex` method.

Reply:
The other variables in GuiSetting have private modifiers, so its a little out of place to make that one variable `DEFAULT_ORDER`  public.

Reply:
I'll just make a getter

Reply:
I think it is okay, if not just implement a getter method since it is a constant, if not just implement a getter method.